prov/efa/src/efa_strerror.c

```diff
@@ -112,7 +128,16 @@ static const char *efa_errno_str(enum efa_errno err)
  */
 const char *efa_strerror(enum efa_errno err)
 {
-	return err >= FI_EFA_ERRNO_OFFSET
-		? efa_errno_str(err)
-		: efa_io_comp_status_str(err);
+	const char *base_err = err >= FI_EFA_ERRNO_OFFSET
+							   ? efa_errno_str(err)
+							   : efa_io_comp_status_str(err);
+	const char *debug_hint = efa_debug_str(err);
+
+	if (debug_hint) {
+		char *err_str = malloc(strlen(base_err) + strlen(debug_hint) + 1);
+		strcpy(stpcpy(err_str, base_err), debug_hint);
```

Comment:
Avoiding the non-standard `stpcpy()` is simple enough:
```suggestion
		strcpy(err_str, base_err);
		strcat(err_str, debug_str);
```

There's also a million other more principled ways to do this, but since this is an error path we probably shouldn't care too much.